pages/inspector/inspector-fix-issues.mdx

```diff
@@ -126,6 +126,47 @@ Once the necessary updates are made in the Tracking Plan, update the issue statu
   />
 </center>
 
+## Resolved issue status
+
+When setting an issue status to _Resolved_ ([see available issue statues](/inspector/inspector-issues-view#issue-status)), you have to specify for what version or what time point the issue should be resolved for.
+
+<center>
+  <img
+    src="/docs/images/inspector/inspector-resolved-issue-status.png"
+    width="600px"
+    alt="Image shows resolved issue options."
+  />
+</center>
+
+The available options are:
+- **Latest release**: The issue will be resolved for the latest release version that Inspector has seen when the issue is marked as resolved.
+- **Next release**: The issue will be resolved for the next release version that is incremented higher than the latest release version that Inspector sees. For example if the latest version is `2.2.1.` and the next version seen that is incremented higher is `2.2.2`, then the issue will be resolved for version `2.2.2` and every version after that.
+- **Specify version**: The issue will be resolved for the release version specified and all versions incremented higher than that.
+- **1 hour**: The issue will be resolved for all release versions seen after 1 hour of marking the issue as resolved.
+- **24 hours**: The issue will be resolved for all release versions seen after 24 hours of marking the issue as resolved.
```

Comment:
This will still mean that versions seen within the 24 hour limit will be validated *after* 24 hours if they appear *again*, as in we do not flag all versions seen within the 24 hours to be ignored forever

Reply:
but versions seen before the user hits resolve are ignored forever?

Reply:
nope, nothing is ignored forever

Reply:
however if an issue exists with a newer version, we are not sending any notifications or doing regressions or anything like that